image/v3/modifying/drawing.md

```diff
@@ -158,9 +158,9 @@ $image = $manager->read('test.png');
 
 // draw a green circle with a white border
 $image->drawCircle(10, 10, function ($circle) {
-    $circle->radius(150); // radius of circle in pixels
-    $circle->background('lightblue'); // background color
-    $circle->border('b53717', 1); // border color & size
+    $circle->setRadius(150); // radius of circle in pixels
+    $circle->setBackgroundColor('lightblue'); // background color
+    $circle->setBorder('b53717', 1); // border color & size
```

Comment:
Please revert. These were correct calls of class `Intervention\Image\Geometry\Factories\CircleFactory::class`.